test/utils/SchemaValidationSpec.scala

```diff
@@ -23,42 +23,48 @@ import org.mockito.ArgumentMatchers
 import org.mockito.Mockito._
 import play.api.libs.json.Json
 import play.api.test.Helpers.{await, defaultAwaitTimeout}
-import testUtils.TestSupport
 
 import scala.concurrent.Future
 
-class SchemaValidationSpec extends TestSupport with MockSchemaRepository {
+class SchemaValidationSpec extends MockSchemaRepository {
 
   def setupMocks(schemaModel: SchemaModel): SchemaValidation = {
     when(mockSchemaRepository.findById(ArgumentMatchers.eq(schemaModel._id)))
       .thenReturn(Future.successful(schemaModel))
-
     new SchemaValidation(mockSchemaRepository)
   }
 
   def setupFutureFailedMocks(schemaModel: SchemaModel): SchemaValidation = {
     when(mockSchemaRepository.findById(ArgumentMatchers.eq(schemaModel._id)))
       .thenThrow(new RuntimeException("Schema could not be retrieved/found in MongoDB"))
-
     new SchemaValidation(mockSchemaRepository)
   }
 
```

Comment:
this file is formatting changes

Reply:
Please use scalafmt plugin which is part of the project for each PR raised

Reply:
okie dokie, updated